app/policies/api_token_policy.rb

```diff
@@ -0,0 +1,37 @@
+class ApiTokenPolicy
+  include ExceptionPolicy
+
+  # ApiTokenPolicy is odd in that it doesn't _have_ a user; we're really just
+  # validating the params, so that will play the role of "user" here. Further,
+  # we'll be determining the api_token from the params, so the second variable
+  # here is not used
+  def initialize(params, record)
+    super(params, record)
+    @params = params
+    @api_token = ApiToken.find_by(access_token: params[:cch])
+  end
+
+  def exists!
+    check(!@api_token.nil?,
+          "Something went wrong with the token (nonexistent)")
+  end
+
+  def not_expired!
+    exists! && check(!@api_token.expires_at || @api_token.expires_at > Time.now,
```

Comment:
> if there is no `expires_at` set on the token, we proceed.

That seems like a mistake, or a hand-wave-y thing we did to make our tests work. It *should* be present for all tokens. Storied:

https://www.pivotaltracker.com/story/show/92351222